tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_provision.c

```diff
@@ -233,13 +242,108 @@ static void oob_auth_set(int test_step)
 {
 	ASSERT_TRUE(test_step < ARRAY_SIZE(oob_auth_test_vector));
 
+	if (test_step > 0) {
```

Comment:
Minor. I would rather try to avoid dependency on items order in the test vector and e.g. add additional field to the test vector.